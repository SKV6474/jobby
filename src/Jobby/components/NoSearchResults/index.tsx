import React from "react";

import { NO_JOBS_IMG } from "../../constants/Images";

import {
  NoSearchContent,
  NoSearchDiv,
  NoSearchImg,
  NoSearchTitle,
} from "./styledComponents";

const NoSearchResults = () => {
  return (
    <NoSearchDiv>
      <NoSearchImg src={NO_JOBS_IMG} alt="No Jobs"></NoSearchImg>
      <NoSearchTitle>No Jobs Found"</NoSearchTitle>
      <NoSearchContent>
        We could not find any jobs. Try other filters.
      </NoSearchContent>
    </NoSearchDiv>
  );
};

export default NoSearchResults;
