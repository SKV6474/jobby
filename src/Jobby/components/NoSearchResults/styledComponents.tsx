import styled from "styled-components";

export const NoSearchDiv = styled.div`
  padding: 40px 0 0;
  display: flex;
  flex-direction: column;
  align-items: center;
  @media screen and (max-width: 768px) {
    padding: 0px 20px 0;
  }
`;

export const NoSearchImg = styled.img`
  height: 300px;
  width: 400px;
  @media screen and (max-width: 768px) {
    height: 200px;
    width: 300px;
  }
`;

export const NoSearchTitle = styled.div`
  font-size: 24px;
  height: 32px;
  font-weight: 500;
  color: ${(props) => props.theme.COLOR};
  margin: 30px 0 25px;
  @media screen and (max-width: 768px) {
    margin: 15px 0 10px;
    font-size: 22px;
    height: auto;
  }
`;

export const NoSearchContent = styled.div`
  height: 24px;
  color: ${(props) => props.theme.CHILD_TEXT_COLOR};
  margin-bottom: 25px;
  text-align: center;
  line-height: 1.5;
  @media screen and (max-width: 768px) {
    margin-bottom: 20px;
    font-size: 18px;
  }
`;
