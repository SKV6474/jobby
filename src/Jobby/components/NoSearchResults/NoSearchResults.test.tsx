import renderer from "react-test-renderer";
import NoSearchResults from ".";

it("renders correctly", () => {
  const tree = renderer.create(<NoSearchResults />).toJSON();
  expect(tree).toMatchSnapshot();
});
