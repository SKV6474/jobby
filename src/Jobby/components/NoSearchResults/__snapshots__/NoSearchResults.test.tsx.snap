// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="sc-beqWaB ehFqut"
>
  <img
    alt="No Jobs"
    className="sc-gueYoa uNjpV"
    src="https://assets.ccbp.in/frontend/react-js/no-jobs-img.png"
  />
  <div
    className="sc-dmqHEX dDHHHW"
  >
    No Jobs Found"
  </div>
  <div
    className="sc-hLseeU cLyjGe"
  >
    We could not find any jobs. Try other filters.
  </div>
</div>
`;
